scripts/gen-tag-table.py

```diff
@@ -335,49 +335,54 @@ def __init__(self):
         self.from_bcp_47_uninherited = None
         # Whether the parser is in a <td> element
         self._td = False
-        # Whether the parser is after a <br> element within the current <tr> element
-        self._br = False
+        # Whether the parser ignores the rest of the current <td> element
+        self._disengaged = False
         # The text of the <td> elements of the current <tr> element.
         self._current_tr = []
 
-    def handle_starttag(self, tag, attrs):
-        if tag == 'br':
-            self._br = True
+    def handle_starttag (self, tag, attrs):
+        if tag == 'a':
+            if self._current_tr and not self._disengaged:
+                self._current_tr[-1] = ''
+                self._disengaged = True
+        elif tag == 'br':
+            self._disengaged = True
         elif tag == 'meta':
             for attr, value in attrs:
                 if attr == 'name' and value == 'updated_at':
-                    self.header = self.get_starttag_text()
+                    self.header = self.get_starttag_text ()
```

Comment:
Yes they are because I copied it from upstream. I think at some point they got reformatted, but I think it's easier to just stick to whatever they have upstream (or we consistently reformat _all_ Python files using a formatter, but that can be done in a separate PR, if necessary).

Reply:
Yes I agree. Or better yet we go upstream and get them reformatted with `ruff format` ;-) In any case if that's just upstream stuff coming in from this release cycle I agree it should get applied to track upstream for ease of diffing. Carry on.

Reply:
+1 for using `ruff`! 😄

Reply:
If we are going to reformat, then it should be formatted in HarfBuzz. The format of the Python scripts is unusual for Python, but it follows the HarfBuzz’s C formatting.